web-ui/src/context/AppContext.js

```diff
@@ -244,7 +244,7 @@ const AppContextProvider = (props) => {
     let currentMonth = new Date().getMonth() + 1;
     let newDate = prevCheckinDate ? new Date(...prevCheckinDate) : new Date();
     if (prevCheckinDate) {
-      let prevCheckinMonth = prevCheckinDate.getMonth() + 1;
```

Comment:
Is this a stray change? Seems unconnected to the member profile card.

Reply:
it's actually preventing a really strange error. I can't explain it yet...but this fixes it.